README.md

```diff
@@ -0,0 +1,97 @@
+# FP-Signature
+
+Fynd platform's signature logic
+
+
+
+FP-Signature is a versatile npm package that provides signature logic for Fynd platform requests. This package supports both CommonJS and ES modules, and it also comes with a web bundle for direct usage in browsers.
+
+## Installation
+
+You can install FP-Signature via npm:
+
+```bash
+npm install @gofynd/fp-signature
+```
+
+## How to Use
+
+### For CommonJS
+
+```javascript
+const { sign } = require("@gofynd/fp-signature");
+```
+
+### For ES Modules
+
+```javascript
+import { sign } from "@gofynd/fp-signature";
+```
+
+### For Browser
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@gofynd/fp-signature"></script>
```

Comment:
Are we publishing this on CDN ? @bhargavprajapatiFynd

Reply:
yes
cdn url provided in README is working